packages/litexa/src/parser/litexa.pegjs

```diff
@@ -916,6 +916,24 @@ pronounce "tomato" as "<phoneme alphabet="ipa" ph="/təˈmɑːtoʊ/">tomato</pho
 
 Note that you must use SSML in the replacement text; the Litexa
 SSML shorthand statements (e.g "<!Bravo>") will not work.
+
+### localized pronunciations
+
+If you plan to publish your skill to multiple locales, you can define
+locale-specific pronunciations for your voiced responses. Simply add
+your locale-specific pronunciation definitions to the Litexa files that
+reside in said locale's Litexa project directory.
+
+::: tip Use a pronounce.litexa file in each locale's Litexa project directory
+If defining locale-specific pronunciations, we recommend creating a
+pronounce.litexa file in each of your locale's Litexa project directories.
+Put all of your locale-specific pronunciation definitions inside of it.
+:::
```

Comment:
Would include the below tip in this paragraph and slightly abbreviate it (since it's somewhat contingent on project setup). Perhaps something like: ... in said locale's Litexa project directory (see the Localization chapter, for further information). We recommend using a standalone file (e.g. `pronounce.litexa`) for pronunciation definitions.